api/routes/graphql/index.js

```diff
@@ -3,13 +3,17 @@ import connect from 'Utils/connect';
 import cookieParser from 'micro-cookie';
 import { ApolloServer } from 'apollo-server-micro';
 import authenticate from 'Middlewares/authenticate';
+import CalendarEventInvite from 'Services/CalendarEventInvite';
 import User from 'Services/User';
 
 const server = new ApolloServer({
   schema,
   context: ({ req }) => ({
     viewer: req.viewer,
-    loaders: { userAvailabilityLoader: User.getAvailabilityLoader() },
+    loaders: {
+      userAvailabilityLoader: User.getAvailabilityLoader(),
```

Comment:
I think this `userAvailabilityLoader` --> `userAvailability` should be done. Then we can do `ctx.loaders.userAvailability.load(key)`

Reply:
makes sense, go ahead refactor it.